java-client/build.gradle.kts

```
@@ -184,12 +184,12 @@ dependencies {
 
     // EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
     // https://github.com/eclipse-ee4j/jsonp
-    api("jakarta.json:jakarta.json-api:2.0.1")
+    api("jakarta.json:jakarta.json-api:2.1.0")
 
     // Needed even if using Jackson to have an implementation of the Jsonp object model
     // EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
     // https://github.com/eclipse-ee4j/parsson
-    api("org.eclipse.parsson:parsson:1.0.0")
+    api("org.eclipse.parsson:parsson:1.1.0")
```

This has to be kept in sync with `jakarta.json`, so let's keep version 1.0.0 for now.

```suggestion
    api("org.eclipse.parsson:parsson:1.0.0")
```